### ISSUE TITLE
Ignore ores below bedrock level

### ISSUE THREAD

**Issue by bendauphinee** [MichaelHillcox/XRay-Mod#125]
It's annoying when I go hunting a block that's below bedrock. Can we filter out ores below something like y3 or y2?

Alternatively, a config that would let us set a y range to x-ray, which would also open it up to being able to hunt for ores that are at a specific depth.

**Comment by MichaelHillcox** [comment]
Why would there be ores below bedrock haha?

**Comment by bendauphinee** [comment]
I dunno if they're below y1-y2, but I've come across a few where I cannot get to them as they're surrounded by bedrock blocks.

**Comment by MichaelHillcox** [comment]
The mod has a hard limit at y=0 so I'm not sure why these blocks would not show...

**Comment by bendauphinee** [comment]
Ok, so, maybe I'm explaining this wrong. I'm at y2 with [this screenshot](https://i.imgur.com/lKO9R2H.png), and that highlighted diamond is 100% inaccessible to me, because it's surrounded by bedrock.

I'd like the option to ignore ores below a set y level, 3 by default maybe, or configurable if it isn't hard to do.

Alternatively, it'd be neat to have a feature that allowed us to set a y range, like y10 - y30, and ignore everything outside that range.

**Comment by MichaelHillcox** [comment]
Yeah, I see what you mean here, I can do this but it'll likely just be hidden in the config options. What version are you on?

**Comment by YokujinKaze** [comment]
you can mine it but you need some other hacks, such as ghosthand

**Comment by bendauphinee** [comment]
At that time of posting, xray-1.15.2-2.2.0

**Comment by ElvenKindH** [comment]
Could the hard limit at y=0 be removed/revamped now?

**Comment by MichaelHillcox** [comment]
For 1.18 it has been, I still don't understand the problem here, nothing spawns below 0? Do you just want ores that you can't access to not show?

**Comment by ElvenKindH** [comment]
It works now somehow. Reinstalled everything and it seems to have fixed it for me at least.
